.github/workflows/feature-build.yaml

```diff
@@ -296,6 +296,23 @@ jobs:
           path: everest-catalog
           token: ${{ secrets.ROBOT_TOKEN }}
 
+      - name: Catalog - update veneer file
+        run: |
+          cd everest-catalog/tools
+          go run . \
+            --veneer-file ../veneer/everest-operator.yaml \
+            --channel fast-v0  \
+            --new-version ${{ env.VERSION }} \
+          cd ..
+          curl -Lo /tmp/opm https://github.com/operator-framework/operator-registry/releases/download/v1.48.0/${OS}-${ARCH}-opm
+          chmod +x /tmp/opm
+          /tmp/opm alpha render-template basic --skip-tls -o yaml < veneer/everest-operator.yaml > catalog/everest-operator/catalog.yaml
```

Comment:
tried to run the FB workflow from this branch and faced
```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 83 60.6M   83 50.4M    0     0   186M      0 --:--:-- --:--:-- --:--:--  186M
100 60.6M  100 60.6M    0     0   204M      0 --:--:-- --:--:-- --:--:--  406M
/home/runner/work/_temp/163a4d30-3236-483f-b4d9-1603bf6f8ff5.sh: line 9: veneer/everest-operator.yaml: No such file or directory
Error: Process completed with exit code 1.
```

putting this comment to prevent merging without fixing the issue